backend/src/routes/teams/management/add_user.rs

```diff
@@ -0,0 +1,110 @@
+use crate::models::entities::teams;
+use crate::utils::app_state;
+use crate::utils::error::Error;
+use crate::utils::jwt::Claims;
+use actix_web::middleware::from_fn;
+use actix_web::{get, post, web, HttpResponse};
+use sea_orm::EntityTrait;
+use serde::{Deserialize, Serialize};
+use serde_json::json;
+use utoipa::ToSchema;
+
+#[derive(Serialize, Deserialize, ToSchema)]
+pub struct AddUserModel {
+    pub username: String,
+}
```

Comment:
Split to separate files/create invitations module. I don't think it has to be submodule of management but it can be